Structure_Engine/Create/SectionProperties/SteelSection.cs

```diff
@@ -41,12 +42,12 @@ public static partial class Create
         /***************************************************/
 
         [Description("Creates a steel I-section from input dimensions. Please note that all units are in S.I., that is meters [m].")]
```

Comment:
Are we able to use quantities in the description?

Reply:
currently not, no, and I am not sure if it would make sense to do so, given that the inputs might vary. Could ofc get rid of this part of the description, but think it might be helpful?

Reply:
The inputs all have the quantities of [length], so is it necessary in the description too?

Reply:
No, think you are right. Will get rid of it